homeassistant/components/group/notify.py

```diff
@@ -66,18 +65,11 @@ async def async_send_message(self, message: str = "", **kwargs: Any) -> None:
         payload: dict[str, Any] = {ATTR_MESSAGE: message}
         payload.update({key: val for key, val in kwargs.items() if val})
 
-        tasks: list[asyncio.Task[bool | None]] = []
         for entity in self.entities:
             sending_payload = deepcopy(payload.copy())
             if (default_data := entity.get(ATTR_DATA)) is not None:
                 add_defaults(sending_payload, default_data)
-            tasks.append(
-                asyncio.create_task(
-                    self.hass.services.async_call(
-                        DOMAIN, entity[ATTR_SERVICE], sending_payload
-                    )
-                )
+            # This starts the call without blocking on the result
```

Comment:
I think that the initial code was wrong, and it should have set `blocking=True`, because we want the "action" to be done when the service call is done. So I would revert this change, and instead add `blocking=True`